test/dynamic_config_test.go

```diff
@@ -263,3 +516,193 @@ func getPodIDs(ctx context.Context, aeroCluster *asdbv1.AerospikeCluster) (map[s
 
 	return pidMap, nil
 }
+
+func updateValue(val interface{}) interface{} {
+	switch val2 := val.(type) {
+	case []string:
+
+	case string:
+
+	case bool:
+
+	case int:
+		return val2 + 1
+	case uint64:
+		return val2 + 1
+	case int64:
+		return val2 + 1
+	case float64:
+		return val2 + 1
+
+	case lib.Stats:
+
+	default:
```

Comment:
Let's remove the nil type. We would know if there is any other type we are missing.